deploy-manage/deploy/self-managed/_snippets/clean-up-multinode.md

```diff
@@ -0,0 +1,9 @@
+Perform the following steps on each node in the cluster:
+
+1. Open `elasticsearch.yml` in a text editor.
+2. Comment out or remove the `cluster.initial_master_nodes` setting.
```

Comment:
> 2. Comment out or remove the `cluster.initial_master_nodes` setting.

Would it make sense to indicate that that setting will only appear in the first node?

Reply:
added `if present` - just want to avoid sending the message that it should be appearing anywhere and it's a bit of a wording challenge.